enterprise_access/apps/api_client/tests/test_constaints.py

```diff
@@ -0,0 +1,6 @@
+"""
```

Comment:
nit: `test_constants.py`, not `test_constaints.py`.

Reply:
🤣 😅